twas-nd/pom.xml

```diff
@@ -33,7 +33,7 @@
     <name>${project.artifactId}</name>
 
     <properties>
-        <azure.apiVersion>2020-06-01</azure.apiVersion>
-        <azure.apiVersion2>2019-06-01</azure.apiVersion2>
+        <git.repo>WASdev</git.repo>
+        <template.pid.properties.url>${artifactsLocationBase}/${project.artifactId}/${git.tag}/config.properties</template.pid.properties.url>
```

Comment:
Same comment as `git.tag` elsewhere in this review.

Reply:
Thanks. Fixed with 1573df4.